src/blueprints/tv2/factories/tv2-atem-video-mixer-timeline-factory.ts

```diff
@@ -45,4 +53,52 @@ export class Tv2AtemVideoMixerTimelineFactory implements Tv2VideoMixerTimelineOb
   private getDownstreamKeyerLayerPrefix(): string {
     return Tv2AtemLayer.DOWNSTREAM_KEYER
   }
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  public createDownstreamKeyerFullPilotTimelineObjects(_blueprintConfiguration: Tv2BlueprintConfiguration): TimelineObject[] {
+    return []
+  }
+
+  public createFullPilotTimelineObjects(_blueprintConfiguration: Tv2BlueprintConfiguration, properties: AtemFullPilotTimelineObjectProperties): TimelineObject[] {
```

Comment:
`FullPilot` is a Viz concept. Atem has no idea what that is and therefore should not know about it.
You are creating a TimelineObject for the PROGRAM, CLEAN and NEXT layers. None of those are `FullPilot` specific.

Reply:
`FullPilot` is gone from the method names.